Test/linters/constructorMatch.dfy.expect

```diff
@@ -0,0 +1,5 @@
+/Users/tobiaja/Documents/dafny/Test/linters/constructorMatch.dfy(10,4): Warning: this branch is redundant 
```

Comment:
This expect file seems incorrect since it contains the current but not the newly desired behavior.